src/bus/bus.c

```diff
@@ -153,9 +164,13 @@ static int bus_log_commit_policy(Bus *bus, const char *action, uint64_t sender_i
 }
 
 int bus_log_commit_policy_send(Bus *bus, uint64_t sender_id, uint64_t receiver_id, NameSet *sender_names, NameSet *receiver_names, Message *message) {
-        return bus_log_commit_policy(bus, "send", sender_id, receiver_id, sender_names, receiver_names, message);
+        return bus_log_commit_policy(bus, "send", "internal", NULL, NULL, sender_id, receiver_id, sender_names, receiver_names, message);
 }
 
 int bus_log_commit_policy_receive(Bus *bus, uint64_t receiver_id, uint64_t sender_id, NameSet *sender_names, NameSet *receiver_names, Message *message) {
-        return bus_log_commit_policy(bus, "receive", sender_id, receiver_id, sender_names, receiver_names, message);
+        return bus_log_commit_policy(bus, "receive", "internal", NULL, NULL, sender_id, receiver_id, sender_names, receiver_names, message);
```

Comment:
TODO item: We could also distinguish between implicit builtin rules and xml rules.

Reply:
Agreed